themes/hugo-tourmaline/layouts/partials/footer.html

```diff
@@ -5,12 +5,12 @@
               {{ with .Param "footer.copyright" }}{{ replace . "{year}" now.Year | markdownify}}{{ end }}<br>
               {{ end }}
               {{ if .Param "footer.support" }}
-              {{ with .Param "footer.support" }}{{ replace . "{year}" now.Year | markdownify}}{{ end }} <a class="rstudioLogo" href="https://www.rstudio.com/"></a>
+              {{ with .Param "footer.support" }}{{ replace . "{year}" now.Year | markdownify}}{{ end }} <a class="rstudioLogo" href="https://www.rstudio.com/" title="RStudio Homepage" aria-label="RStudio Homepage"></a>
```

Comment:
I think we should only set `aria-label` here. Title is for tooltips and I don't think we care about it.

[This issue](https://github.com/nvaccess/nvda/issues/7841) suggests that some screen readers will read both, which seems annyoing.

Reply:
Done. WAVE seems happy with it, too.